base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm_single_temp.opi

```diff
@@ -0,0 +1,315 @@
+    <background_color>
+      <color name="ISIS_Label_Background" red="240" green="240" blue="240" />
+    </background_color>
+    <border_color>
+      <color name="ISIS_Border" red="0" green="0" blue="0" />
+    </border_color>
+    <border_style>0</border_style>
+    <border_width>1</border_width>
+    <enabled>true</enabled>
+    <font>
+      <opifont.name fontName="Segoe UI" height="14" style="1" pixels="false">ISIS_Header2_NEW</opifont.name>
+    </font>
+    <foreground_color>
+      <color name="ISIS_Standard_Text" red="0" green="0" blue="0" />
+    </foreground_color>
+    <height>37</height>
+    <horizontal_alignment>0</horizontal_alignment>
+    <name>Label_1</name>
+    <rules />
+    <scale_options>
+      <width_scalable>true</width_scalable>
+      <height_scalable>true</height_scalable>
+      <keep_wh_ratio>false</keep_wh_ratio>
+    </scale_options>
+    <scripts />
+    <show_scrollbar>false</show_scrollbar>
+    <text>$(NAME)</text>
+    <tooltip></tooltip>
+    <transparent>false</transparent>
+    <vertical_alignment>1</vertical_alignment>
+    <visible>true</visible>
+    <widget_type>Label</widget_type>
+    <width>776</width>
+    <wrap_words>true</wrap_words>
+    <wuid>6c52603b:18311cdccfc:-6e09</wuid>
+    <x>6</x>
+    <y>-24</y>
+  </widget>
+  <widget typeId="org.csstudio.opibuilder.widgets.Label" version="1.0.0">
+    <actions hook="false" hook_all="false" />
+    <auto_size>false</auto_size>
+    <background_color>
+      <color red="255" green="255" blue="255" />
+    </background_color>
+    <border_color>
+      <color name="Invalid" red="255" green="0" blue="255" />
+    </border_color>
+    <border_style>0</border_style>
+    <border_width>1</border_width>
+    <enabled>false</enabled>
+    <font>
+      <opifont.name fontName="Segoe UI" height="9" style="0" pixels="false">Default</opifont.name>
+    </font>
+    <foreground_color>
+      <color name="Major" red="255" green="0" blue="0" />
+    </foreground_color>
+    <height>37</height>
+    <horizontal_alignment>1</horizontal_alignment>
+    <name>Label_2</name>
+    <rules />
+    <scale_options>
+      <width_scalable>true</width_scalable>
+      <height_scalable>true</height_scalable>
+      <keep_wh_ratio>false</keep_wh_ratio>
+    </scale_options>
+    <scripts>
+      <path pathString="EmbeddedPy" checkConnect="false" sfe="false" seoe="false">
+        <scriptName>EmbeddedScript</scriptName>
+        <pv trig="true">$(PV_ROOT):A01:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A02:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A03:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A04:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A05:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A06:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A07:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A08:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A09:ACTIVE</pv>
+        <pv trig="true">$(PV_ROOT):A010:ACTIVE</pv>
+      </path>
+    </scripts>
```

Comment:
This has become corrupt in some way, it's an embedded python script without a script, which causes CSS to be unable to edit it...

In any case, I think we can actually make this check better on this OPI: instead of just checking that *any* sensor is configured (which the main eurotherm OPI does), we should check whether *this specific* sensor is configured.